ts/store/reducers/entities/readTransactions.ts

```diff
@@ -10,11 +11,16 @@ export const isReadTransaction = (state: GlobalState, txid: number) =>
   state.entities.transactionsRead[txid] !== undefined;
 
 // filter only unread transactions to account for the residual number
-export const getUnreadTransactions = (state: GlobalState) =>
+const getUnreadTransactions = (state: GlobalState) =>
   pot.map(state.wallet.transactions.transactions, txs =>
     values(txs).filter(_ => _ !== undefined && !isReadTransaction(state, _.id))
   );
 
+export const getUnreadTransactionsSelector = createSelector(
```

Comment:
Useless memoization

Reply:
So I'm afraid I'm missing your point.
I understood you wanted memoization for the getUnreadTransactions function but now I'm not sure about what you were talking about, could you please highlight the point?

Reply:
The point is simple: recalculate the "**unreadTransactions**" only when the "**transactions**" list changes.